lte/cloud/go/services/subscriberdb/storage/digest.go

```diff
@@ -191,14 +190,15 @@ func GetOutdatedNetworks(l DigestLookup, lastUpdatedBefore int64) ([]string, err
 	return networks, nil
 }
 
-// GetAllNetworks returns all networks currently stored.
+// GetAllNetworks returns all unique networks currently stored.
 func GetAllNetworks(l DigestLookup) ([]string, error) {
 	digestInfos, err := l.GetDigests([]string{}, clock.Now().Unix())
 	if err != nil {
 		return nil, err
 	}
 	networks := digestInfos.Networks()
-	return networks, nil
+	networksUniq := funk.UniqString(networks)
+	return networksUniq, nil
 }
 
 type digestInfo struct {
```

Comment:
digestInfo and its fields should all be public -- since we're making DigestInfos public